.github/actions/setup-rocm/action.yml

```diff
@@ -68,7 +54,7 @@ runs:
         fi
 
     - name: Runner diskspace health check
-      uses: pytorch/pytorch/.github/actions/diskspace-cleanup@main
+      uses: ./.github/actions/diskspace-cleanup
```

Comment:
This is a temporary change in this PR, required to test the changes to `diskspace-cleanup/action.yml` in this PR. We will revert this change right before merging this PR.